python/tk_multi_workfiles/file_list/file_group_widget.py

```diff
@@ -93,7 +94,13 @@ def set_item(self, model_idx):
         if search_status == FileModel.SEARCHING and not idx_has_children:
             search_msg = "Searching for files..."
         elif search_status == FileModel.SEARCH_COMPLETED and not idx_has_children:
-            search_msg = "No files found!"
+            templates = work_area.get_missing_templates()
+            if not work_area.are_settings_loaded():
+                search_msg = "The Shotgun File Manager hasn't been setup."
```

Comment:
Just change it here for now, we can update ZD docs in the future... I think there would need to be more than just the info.yml updated.